src/components/postHtmlRenderer/videoThumb.tsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
-import { View, ImageBackground } from 'react-native';
+import { View, ImageBackground, TouchableHighlight } from 'react-native';
```

Comment:
also here about TouchableHighlight, if we using RN and it is working ok, we should consider removing `react-native-gesture-handler` package.